woocommerce/woocommerce-ios
# Dynamic Dashboard: Remove the feature flag

Closes: #12562 


## Description

This PR removes the feature flag `dynamicDashboard` to enable the feature. This also comes with a bit of cleaning up for the UserDefaults keys used for showing/hiding store onboarding and Blaze cards, as well as the configure in Settings screen for store onboarding.

There are a few more to-dos after this (which I added to the main task list of the project):
- Handle the restoring of the Blaze card after creating a new campaign.
- Removing stale feature flags for the store onboarding card.
- Clean up files of the legacy store stats flow.

Pardon the large number of changes, most of them are from removing tests 🙏 

## Testing instructions

- Build the app and log in to a store.
- Do a smoke test following the testing plan to confirm that everything looks good. Double-check the visibility of store onboarding and Blaze cards.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/aa0413f2-9622-49e0-a59a-e37cd83d0542



---
